src/components/PeopleDropdown/PeopleDropdown.tsx

```diff
@@ -0,0 +1,61 @@
+import debounce from 'lodash.debounce';
+import React, { useCallback } from 'react';
+
+interface Props {
+  query: string;
+  isHide: boolean;
+  delay: {
+    mainDelay: number,
+    blurDelay: number,
+  };
+  setQuery: (value: string) => void;
+  setIsHide: (value: boolean) => void;
+  applyQuery: (value: string) => void;
+}
+
+export const PeopleDropdown: React.FC<Props> = ({
+  query,
+  isHide,
+  delay,
+  setIsHide,
+  setQuery,
+  applyQuery,
+}) => {
+  const handleClickHide = () => {
+    return isHide ? setIsHide(false) : setIsHide(true);
+  };
+
+  const { mainDelay, blurDelay } = delay;
+
+  const applyHide = useCallback(
+    debounce(setIsHide, mainDelay),
+    [],
+  );
+
+  const handleQueryChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setQuery(event.target.value);
+    applyQuery(event.target.value);
```

Comment:
is it necessary to invoke `applyQuery`?

I think its enough to `setQuery`

Reply:
yes i really need that cuz without it filtration doesn't work(